firefox/makotokato-riscv64-support-and-zenithal-backported.patch

```diff
@@ -2075,25 +2075,25 @@ Subject: [PATCH] More webrtc fix. But this isn't built yet.
 # #elif defined(__riscv) && __riscv_xlen == 64
 # #define ARCH_CPU_RISCV 1
 # #define ARCH_CPU_64_BITS 1
-diff --git a/third_party/libwebrtc/webrtc/typedefs.h b/third_party/libwebrtc/webrtc/typedefs.h
-index 60095a926faa2..2b9ca31d6e47c 100644
---- a/third_party/libwebrtc/webrtc/typedefs.h
-+++ b/third_party/libwebrtc/webrtc/typedefs.h
-@@ -120,6 +120,14 @@
- #define WEBRTC_ARCH_32_BITS 1
- #define WEBRTC_ARCH_BIG_ENDIAN
- #define WEBRTC_BIG_ENDIAN
-+#elif defined(__riscv) && __riscv_xlen == 64
-+#define WEBRTC_ARCH_64_BITS
-+#define WEBRTC_ARCH_LITTLE_ENDIAN
-+#define WEBRTC_LITTLE_ENDIAN
-+#elif defined(__riscv) && __riscv_xlen == 32
-+#define WEBRTC_ARCH_32_BITS
-+#define WEBRTC_ARCH_LITTLE_ENDIAN
-+#define WEBRTC_LITTLE_ENDIAN
- #elif defined(__pnacl__)
- #define WEBRTC_ARCH_32_BITS
- #define WEBRTC_ARCH_LITTLE_ENDIAN
+#diff --git a/third_party/libwebrtc/webrtc/typedefs.h b/third_party/libwebrtc/webrtc/typedefs.h
+#index 60095a926faa2..2b9ca31d6e47c 100644
+#--- a/third_party/libwebrtc/webrtc/typedefs.h
+#+++ b/third_party/libwebrtc/webrtc/typedefs.h
+#@@ -120,6 +120,14 @@
+# #define WEBRTC_ARCH_32_BITS 1
+# #define WEBRTC_ARCH_BIG_ENDIAN
+# #define WEBRTC_BIG_ENDIAN
+#+#elif defined(__riscv) && __riscv_xlen == 64
+#+#define WEBRTC_ARCH_64_BITS
+#+#define WEBRTC_ARCH_LITTLE_ENDIAN
+#+#define WEBRTC_LITTLE_ENDIAN
+#+#elif defined(__riscv) && __riscv_xlen == 32
+#+#define WEBRTC_ARCH_32_BITS
+#+#define WEBRTC_ARCH_LITTLE_ENDIAN
+#+#define WEBRTC_LITTLE_ENDIAN
+# #elif defined(__pnacl__)
+# #define WEBRTC_ARCH_32_BITS
+# #define WEBRTC_ARCH_LITTLE_ENDIAN
```

Comment:
Please just remove merged parts of the patch, and analyze the unmerged conflicts. Don't just comment them out.